source/GooglePlayServicesProject.cshtml

```diff
@@ -143,19 +143,34 @@
       }
   </ItemGroup>
 
+  <!--
```

Comment:
Probably don't need to commit this file.

Reply:
`JavaSourceJar`and `JavaDocJar` parts are commented out, but `if` contains artifact list of the artifacts that cause `java-source-utils.jar`to fail. Thus I added it to the PR. I will work on issue as soon as .NET6 migration and template sets are done.